ReactGoStat/README.md

```diff
@@ -0,0 +1,25 @@
+## ReactGoStat
+### In Early Development
```

Comment:
I would like to have some more general information about the setup of the library. It doesn't have to be super specific, but it should be enough for me to get a clear overview of the structure of the project.